tests/unit_tests/pyanaconda_tests/modules/storage/test_module_storage.py

```diff
@@ -1543,17 +1544,29 @@ def test_lvm_devices_file(self, exists_mock):
         storage = Mock(devices=[dev1, dev2], devicetree=Mock(_hidden=[dev3, dev4]))
 
         with tempfile.TemporaryDirectory() as tmp:
-            # lvm devices file: disabled
+            # lvm devices file: disabled, image installation: False
             with patch("pyanaconda.modules.storage.installation.HAVE_LVMDEVICES", new=False):
+                patched_conf.target.is_image = False
+                WriteConfigurationTask._write_lvm_devices_file(storage, tmp)
```

Comment:
Line 1550 till line 1555 is exactly the same like like 1560 till 1565. 

I am slightly confused here. When the conf file mock has is_image = False, the lvmdevices_add should be called, no? 

If these are expected to be the same, shall we create a helper function for the checks?

Reply:
The difference is whether the LVM devices feature is available or not. Here we have normal installation, but the feature is not available so `lvmdevices_add` shouldn't be called.